velox/functions/sparksql/tests/DateTimeFunctionsTest.cpp

```diff
@@ -16,11 +16,20 @@
 
 #include "velox/common/base/tests/GTestUtils.h"
 #include "velox/functions/sparksql/tests/SparkFunctionBaseTest.h"
+#include "velox/type/Timestamp.h"
 #include "velox/type/tz/TimeZoneMap.h"
 
 namespace facebook::velox::functions::sparksql::test {
 namespace {
 
+std::string timestampToString(Timestamp ts) {
+  std::tm tm;
+  TimestampToStringOptions options;
+  options.mode = TimestampToStringOptions::Mode::kFull;
+  Timestamp::epochToUtc(ts.getSeconds(), tm);
+  return Timestamp::tmToString(tm, 0, options);
```

Comment:
This method does not exist on the main branch now. Maybe Timestamp::tsToStringView is what's needed here.

Reply:
Thank you!